python/nav/bin/navclean.py

```diff
@@ -44,19 +45,20 @@ def main():
         expiry = args.datetime
 
     connection = nav.db.getConnection('default', 'manage')
-    deleters = get_selected_deleters(args, connection)
+    cleaners = get_selected_cleaners(args, connection)
 
-    deleted = False
-    for deleter in deleters:
+    cleaned = False
+    for cleaner in cleaners:
         try:
-            count = deleter.delete(expiry, dry_run=not args.force)
+            count = cleaner.clean(expiry, dry_run=not args.force)
             if not args.quiet:
                 print(
                     "Expired records in {table}: {count}".format(
-                        table=deleter.table, count=count if count is not None else "N/A"
+                        table=cleaner.expiry_type,
+                        count=count if count is not None else "N/A",
```

Comment:
Supposed to still be called `table` here?

Reply:
Yes and no.  As mentioned in @johannaengland's comments, the naming isn't really 100% thought out.  The table name and the command line argument name are being conflated, because there is now more than one type of operation against a table.

Reply:
new naming is good